### DISCUSSION TITLE
The Attunements on ChromieCraft: proposals

### DISCUSSION THREAD

**Discussion by Zalausthra** [chromiecraft/chromiecraft#2188] [General]
**The Attunements on ChromieCraft: proposals**
Hello. In this post I'm going to talk about my idea regarding the introduction and reintroduction of Attunements for Classic content. I'll do a general overview under the roleplaying point of view, marked in _cursive_, and one under the technical one, case by case.
Let's start by explaining the reason for Attunements. Why ask anything of someone who wants to enter a raid? I'll explain it in role-playing terms, while proposing the motivation according to Cromie and in sync with the plot of our server.

_Following the reports of the dungeons, supported by the adventurers, Cromie observes that the mysterious dark energies are not only unaltered, but seem to intensify at historical points of great importance.
Cromie fears that repeated expeditions into the past may damage the containment structures of the unchanging timeline, causing them to collapse. If this happened, creatures defeated in the past era would return to the main time stream: the fallen would rise again, haunting everyone. Since this is not a natural phenomenon, these experiences would be quite alarming to those in the main stream. One would no longer have the absolute certainty of seeing a sunrise and a sunset. The natural order of things would be shattered. Chaos would reign supreme.
Aware of this apocalyptic scenario, and in order to stress as little as possible the structures of the generated temporal ramification, the Ambassador of the Bronze Dragonflight asks the Champions of the Alliance and the Horde to tune in to the historical events, manifested just before the temporal nexuses._

So let's proceed with all the Classic content for which to propose an attunement.

**Upper Blackrock Spire**
The first temporal nexus will be located in the Upper Blackrock Spire. 

_Historically, to gain access to the upper floors of the Blackrock Mountain, the adventurers had to open a very heavy stone door with a ring lock that was impossible to break into, let alone blow up. So, the champions ventured into the lower levels of the ramparts, in search of the coveted ring key. With the help of a red dragon in disguise, they succeeded._

That same door at the top of the access ramp, since patch 3.0.8, is always open, so it is not required to obtain the [Seal of Ascension](https://wowgaming.altervista.org/aowow/?item=12344). So my idea is to physically close this door and reintroduce the Seal requirement to open it. This will make the whole expedition slower as players will also have to explore the lower level of the Spire upon release of Phase B, rather than going directly upstairs. 

**The Molten Core**
The second temporal nexus will be the Molten Core, home of the irascible and powerful Ragnaros, Elemental Lord of Fire. The event manifested prior to the incursion into the seething bowels of the Rocky Mountains concerns the attunement of the Champions with the Molten Core. 

_In the past, it was the Elf Lothos Riftwalker who bound the souls of the adventurers to the rocks of the Molten Core, so that they could break through directly from the Grinding Quarry. In order to weave the intricate teleportation spell, Lothos asked for a fragment of stalagmite taken from the hottest and most remote point of the Blackrock Depths, where the air itself seemed to be on fire._

Officially, this would not be an Attunement to enter the Molten Core, but rather a convenience. Imagine forty players (official version) having to traverse an entire dungeon, either in the long or contracted form, whenever they wish to support this raid. Absolutely inhumane. Which is exactly why Blizzard introduced the ["Attunement to the Core"](https://wowgaming.altervista.org/aowow/?search=Attunement+ot+the+Core) quest, available from [Lothos Riftwalker](https://wowgaming.altervista.org/aowow/?npc=14387)
Now, since CromieCraft has a personalized storyline, in line with the one proposed for the Attunements, and since each of us must support this quest to enter the Molten Core immediately, the introduction of this requirement would be irrelevant in itself. However, in accordance with the proposed plot, it is mostly necessary as a hook to the subsequent attunements.
Ultimately, players who wish to pass through the portal to the Molten Core at the end of the Blackrock dungeon will be required to complete the quest "Attunement ot the Core". I also bring to attention a second entrance, physically identified in the window of the Lothos chamber, on the opposite side of the quarry entrance. For the sole purpose of confirming this possibility, I have personally verified and, as you can see from the system message shown [here](https://ibb.co/GFP9PZ8), the Molten Core is accessible in raid mode from this window. Therefore, in order to avoid probable abuse, it too should be subject to the same requirement.
The npc Lothos Riftwalker still provides the above quest.

**The Blackwing Lair**
_The third temporal nexus is identified in the Blackwing Lair. Shortly after banishing Ragnaros to the Firelands, the Horde and Alliance Champions were made aware of another evil lurking in the Blackrock Mountain, an evil known as Nefarian, son of the ferocious Deathwing.
On the way to the Blackrock Spire, the adventurers encountered the Quartermaster of the Scarred Shield Legion, the division of Blackrock Orcs commanded by Warchief Rend Blackhand. Following the clash between the two forces, the Champions found a letter from the fallen Orc and read its contents, discovering a way to break into the Blackwing Lair, home of Nefarian and his followers.
Crossing the Blackrock Spire, the adventurers slaughtered Rend Blackhand’s minions, including the latter, pouring their untamed fury especially on General Drakkisath, the dragonid guarding the 'key' to the Blackwing Lair. After placing their hand on the glowing orb behind him, the Champions received the Mark of Drakkisath. With it they could use the Orb of Command, located at the end of the entrance corridor to the Spire, to teleport directly into the infamous Lair._

[Blackhand’s Command](https://wowgaming.altervista.org/aowow/?item=18987) is still obtainable and the generated quest can be sustained. Again, this would not be an actual attunement, but rather a convenience converted to Attunement.
Therefore, the requirement to complete the ["Blackhand’s Command"](https://wowgaming.altervista.org/aowow/?quest=7761) quest will have to be introduced to anyone who wants to cross the portal into the Upper Blackrock Spire.

**Ahn’Qiraj**
_Shortly after Hakkar's defeat at Zul'Gurub, an ancient evil awoke in the bosom of Silitus: fierce insectoids known as Silithids, loyal to the Qiraji Lords, swarmed into every corner of the desert, quickly conquering the outposts of resistance.
Horde and Alliance were soon informed of the insectoid threat and organized something unprecedented: a combined force to repel the Qiraji invasion. Varok Saurfang, an experienced Orc warrior, got the command of the campaign.
The army of [Varok Saurfang](https://wowgaming.altervista.org/aowow/?object=180866), Might of Kalimdor Commander and brother of the legendary Broxigar Saurfang, managed to drive out the invaders, but it soon became clear that the enemy would not stop and would eventually overwhelm the combined forces through attrition.
Thus, the champions of Azeroth and a portion of the [Might of Kalimdor](https://wowgaming.altervista.org/aowow/?search=Might+of+Kalimdor#npcs), marched to Ahn'Qiraj, the titanic city under which dwelt the Old God C'thun, the true puppeteer behind the Qiraji forces.
Centuries earlier, C'thun tried to recapture the surface by taking advantage of the Qiraji's awakening. Yet the resistance of Archdruid Fandral Staghelm and the Dragon Aspects managed to drive the invaders back into the titanic city, sealing the entrance with a mighty barrier of enchanted stone: the Scarab Wall. Looking forward to a brighter future, where another Strike Force would end C'thun's rule, Anachronos, son of the Bronze Dragon Aspect, forged two powerful artifacts: [The Scepter of Shifting Sands](https://wowgaming.altervista.org/aowow/?item=21175), entrusted to Fandral, and [The Scarab Gong](https://wowgaming.altervista.org/aowow/?object=180718). If rang, the latter would break the spell protecting the Wall and the Gates of Ahn'Qiraj would be reopened.
However, that same Scepter was destroyed and its pieces scattered around the world. It was not easy to collect them all, but the Champions of Azeroth succeeded. However, the coalition of the Alliance and the Horde could not open the gates: behind them, an unclean quantity of Qiraji was waiting for that act to pour on the resistance. In order to counterattack, the combined Alliance and Horde military force would have to obtain War Effort, equipping and feeding the new troops as best they could. Therefore, it would be necessary to gather a large number of supplies - food, bandages, metals, hides, and herbs. Five days later, when the required supplies arrived in the Silitus desert by air, the Scepter Bearer of the Shifting Sands finally [bang the gong](https://wowgaming.altervista.org/aowow/?quest=8743).
A [Black Qiraji Resonating Crystal](https://wowgaming.altervista.org/aowow/?item=21176) emerged from the sands, within which could be glimpsed the outline of a Qiraji... dark, like its container. It was a more than unique specimen. The crystalline formation shattered and the huge insectoid inside came to life, bowing its back in submission to the Scepter Bearer. He mounted it, raising his scepter under the shouts and cries of all those gathered.
And just in that pleasant moment of jubilation, the dreadful hour struck.
The gates of Ahn'Qiraj opened wide, beginning the Ten Hour War.
A frantic torrent of screeching sylitids poured out of the Scarab Wall, overwhelming the Might of Kalimdor.
While the battle raged on, large floating obelisks called [Resonating Crystal Formation](https://wowgaming.altervista.org/aowow/?object=180810) appeared in the rest of Silithus and in the regions of Tanaris, Barrens, Darkshire, Thousand Needles and Un'Goro Crater. The dark magic of [these crystals](https://wowpedia.fandom.com/wiki/Resonating_Crystal_Formation) was able to control the minds of those in their immediate vicinity, calling forth Silithids and [Anubiasath Warbringer](https://wowgaming.altervista.org/aowow/?search=Anubisath+Warbringer). 
Nine hours after the war began, the Might of Kalimdor Commander gathered the armies of the Horde and Alliance outside the Cenarian Fortress to march on Ahn'Qiraj. There was only one way to stop the continuing insectoid invasions on Kalimdor: defeat their General, Rajaxx.
Reinvigorated by the newcomers, the Might of Kalimdor was able to overwhelm the invaders, killing their General and dealing a severe blow to the enemy. As a tribute to the efforts of those Champions, [Jonathan the Revelator](https://wowgaming.altervista.org/aowow/?npc=15693), Guardian of the Scarab Gong, and spokesman for the Timeless One, donated powerful weapons in order to support the Champions in their fight against C'thun.
Without giving the Qiraji a chance to recover, Varok Saurfang divided the army into two parts: one force would march on the Ruins of Ahn'Qiraj, keeping the enemy busy on the surface; the rest would descend the jaws of madness, clashing with the Ancient God in the flesh.
On that day, the factions involved lost countless lives. In the end, however, the Champions of Azeroth prevailed over C'thun and his minions._

My idea is to reintroduce the War Effort world event, as well as the Ten Hour War.

_In the past, Npc Commander Stronghammer, Alliance Ambassador in the Orgrimmar Valley of Spirits, and same for the Npc General Zog, Horde Ambassador in Ironforge Military Ward, required a considerable amount of supplies._

In addition, both Zog and Stronghammer make use of a number of Npc's in both Orgrimmar Valley of Spirits and Ironforge Military Ward, available for the duration of the event. These subordinates accepted supplies in exchange for [Horde](https://wowgaming.altervista.org/aowow/?item=21438), and [Alliance Commendation Signet](https://wowgaming.altervista.org/aowow/?item=21436), depending on which faction they belonged to. If the supplies were delivered for the first time, they also received the box of [Ahn'Qiraj War Effort Supplies](https://wowgaming.altervista.org/aowow/?item=21511); returning to their respective Ambassadors, it was possible to exchange the Commendation Signet for other boxes of the same type.
Once the amount of supplies desired by both factions had been reached, they had to be physically mobilized, in purely imaginary terms, taking into account the physical distance between Orgrimmar, Ironforge and Silithus. Historically, there is talk of five days of travel aboard zeppelins.
And now we come to the main course. In Classic and Vanilla, the first player to ring the Scarab Gong, as well as all other bearers of this artifact, would receive the title [“Scarab Lord”](https://wowgaming.altervista.org/aowow/?achievement=416) and the legendary mount of [Black Qiraji Battle Tank](https://wowgaming.altervista.org/aowow/?spell=26656).
The Ten Hour War was fought on several fronts: at the Gates of Ahn'Qiraj and in several regions of Kalimdor, including Silithus. Regarding the latter, killing a Colossus in three separate Hives, for example [Colossus of Regal](https://wowgaming.altervista.org/aowow/?npc=15741), activates three separate repeatable quests. These missions were deactivated, like the others relevant to the event, at the end of Ten Hour War.
At the end of the World War, all those who possessed the Scepter could benefit from the epic weapons offered by the quest ["The Treasure of the Timeless"](https://wowgaming.altervista.org/aowow/?quest=8745), which can be started by the Watcher Jonathan the Revelator, next to the Scarab Gong.
Unlike previous raids, this attunement would serve to introduce the Ahn'Qiraj raid, allowing adventurers to tune into that given time nexus, through the War Effort event and the subsequent related Ten Hour War. Of course, during this timeframe, the Scarab Wall must be raised, deactivating it at the end of the 10 hours following the bang of Gong.

**Conclusions**
As you can see from each of the links, related to the database 'Aowow', all the contents of this post are still present in 3.3.5a, many of which should be activated again, similarly to the Scarab Wall.
In my humble opinion, basing this thesis on the role-playing game plot proposed at the beginning of this post, I think it is necessary to give maximum coherence to the story, leaving the mechanics of the world event unchanged, with the exception of the overall supplies required by the Ambassadors in the War Effort world event.
Regarding these resources, considering the original numbers, they would not be suitable for the reality of ChromieCraft, even in the most desirable case estimated for the bracket of interest. For this reason, I propose to resize the overall supplies required by both NPCs, according to:
- the population of the server, estimated between the bracket of the Blackwing Lair and the release of the one related to Ahn'Qiraj;
- the fact that, since each elite guild is engaged in the development of the questline related to the Scepter of the Shifting Sands, the remaining population of ChromieCraft will have to deal with finding supplies to feed the Alliance and Horde supplies.
This is the end of the Classic content attunements, but remember: to remain faithful to the plot and consistent with your choices, you must also respect the Burning Crusade attunements.

**Comment by 55Honey** [comment]
Hello @Zalausthra  !
Thank you for the detailled post. We've looked into the upcoming parts already and we will continue to work on the historical part of the raids. 

There are a lot of worthy suggestions in this and we will touch them one after another as soon as they become relevant for our progression.

Again, thanks for the Research.

**Reply by Zalausthra** [reply]
Hello, @55Honey !
Thank you for your appreciation of my work. In your words, I saw the hope to raise the gaming experience of us players to an even higher level. I can only expect this, reading the announcement, a few days ago, about the first raid. 
In particular, I appreciated the debuff that will afflict all players who enter Upper Blackrock Spire. As I see it, this is another cautionary tool adopted by Chromie that, aware of the high performance of time-travelers, will apply this time spell to reduce their performance and adapt it to the past era. Assuming and granting that I have correctly understood the effect of the infamous debuff. 
In any case, I want thanking the ChromieCraft' staff for its efforts!!!
